integration-tests/src/main/java/io/quarkiverse/jackson/jq/it/MyFunction.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkiverse.jackson.jq.it;
+
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import io.quarkiverse.jackson.jq.JqFunction;
+import net.thisptr.jackson.jq.Expression;
+import net.thisptr.jackson.jq.Function;
+import net.thisptr.jackson.jq.PathOutput;
+import net.thisptr.jackson.jq.Scope;
+import net.thisptr.jackson.jq.Version;
+import net.thisptr.jackson.jq.exception.JsonQueryException;
+import net.thisptr.jackson.jq.path.Path;
+
+@JqFunction({ "myFunction/1", "myFunction/2" })
```

Comment:
I've created a custom annotation here to avoid clashes with the annotation provided by jackson-jq